backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/scheduler/PortalPrototypePublishingScheduler.java

```diff
@@ -16,7 +16,7 @@
  */
 @Component
 @Slf4j
-@Profile({ "production" })
+@ConditionalOnProperty("publish.portal-prototype.enabled")
```

Comment:
What are your arguments to change this? Two thoughts:
1. Do we know that we may need to disable this job on-demand? This would be a good argument for using such a property. Nevertheless I think we are deviating of the usage we had till now in regards of activating something depending on environments
2. Should we had a default value here? I think spring set "missing" property for conditionals to `false` but not 100% sure. Should we for visibility just use the `matchIfMissing = false` ?

Reply:
With using a profile it was hard to be able to create an integration test as that test would then need to use that profile, but that created other problems. In this way I was able to just enable this feature in the integration test without also taking eg. the prod database config.

The default for a missing property is false, and that is also what i would expect when i would be reading that code without any knowledge about that annotation so I think it's clear enough this way?